src/groups/mqb/mqbc/mqbc_clusterstate.cpp

```diff
@@ -807,6 +807,17 @@ void ClusterState::DomainState::adjustQueueCount(int by)
     }
 }
 
+void ClusterState::DomainState::adjustOpenedQueueCount(int by)
+{
```

Comment:
```suggestion
{
    // Thread: CLUSTER dispatcher
```

or longer

```suggestion
{
    // Executed by the CLUSTER dispatcher thread
```

This comment is needed to justify safety of change of `int d_numOpenedQueues`